Kickstarter-iOS/Views/Controllers/CreatePasswordViewController.swift

```diff
@@ -48,13 +69,66 @@ final class CreatePasswordViewController: UITableViewController {
     self.tableView.registerCellClass(SettingsTextInputCell.self)
   }
 
+  override func viewDidAppear(_ animated: Bool) {
+    super.viewDidAppear(animated)
+
+    self.viewModel.inputs.viewDidAppear()
+  }
+
   // MARK: - Styles
 
   override func bindStyles() {
     super.bindStyles()
 
     _ = self.tableView
       |> settingsGroupedTableViewStyle
+      |> \.allowsSelection .~ true
+  }
+
+  // MARK: - Binding
+
+  override func bindViewModel() {
+    super.bindViewModel()
+
+    self.viewModel.outputs.accessibilityFocusValidationLabel
+      .observeForUI()
+      .observeValues { [weak self] _ in
+        UIAccessibility.post(notification: .layoutChanged, argument: self?.groupedFooterView?.label)
+    }
+
+    self.viewModel.outputs.newPasswordConfirmationTextFieldBecomeFirstResponder
+      .observeForUI()
+      .observeValues { [weak self] in
+        self?.newPasswordConfirmationTextField?.becomeFirstResponder()
+    }
+
+    self.viewModel.outputs.newPasswordConfirmationTextFieldResignFirstResponder
+      .observeForUI()
+      .observeValues { [weak self] in
+        self?.newPasswordConfirmationTextField?.resignFirstResponder()
+    }
+
+    self.viewModel.outputs.validationLabelIsHidden
+      .observeForUI()
+      .observeValues { [weak self] isHidden in
+        self?.groupedFooterView?.isHidden = isHidden
+    }
+
+    self.viewModel.outputs.validationLabelText
+      .observeForUI()
+      .observeValues { [weak self] text in
+        UIView.performWithoutAnimation {
+          self?.tableView.beginUpdates()
+          self?.groupedFooterView?.label.text = text
```

Comment:
I wonder if it would be a better practice to create a public text var on the GroupedFooterView class and set the label text there, this way we wouldn't need to modify a label's property from outside its class (same principle of private IBOutlets). Something like: 
```
public var text: String? {
    didSet {
      _ = self.label
    |> \.text .~ self.text
    }
  }
```

Reply:
I was initially considering this approach but ended up going with allowing accessing the `label` directly in order to follow UIKit patterns that apply similarly for `UITableViewCell` (which exposes `textLabel` and `detailTextLabel`, or even `UITableViewHeaderFooterView` that exposes `textLabel` and `detailTextLabel` (unfortunately these didn't work well with custom auto layout so they're just there unused).